app/Presenters/AssetPresenter.php

```diff
@@ -268,8 +268,11 @@ public static function dataTableLayout()
                 "searchable" => true,
                 "sortable" => true,
                 "switchable" => true,
-                "title" => ($field->field_encrypted=='1') ?'<i class="fa fa-lock"></i> '.$field->name : $field->name,
-                "formatter" => "customFieldsFormatter"
+                "title" => $field->name,
+                'formatter' => 'customFieldsFormatter',
```

Comment:
It's the tiniest of minor nits, but you're switching from double-quotes to single-quotes.

Reply:
Good catch, thanks @uberbrady - really they should all be single quotes there, but it's weird to have them mixed. Switching to double for this PR, can clean that up, style-wise later.

Reply:
Can you re-check now please? My changes are pushed.